lang/cs/Org.Apache.REEF.Network/Group/Config/GroupCommConfigurationOptions.cs

```diff
@@ -44,7 +44,7 @@ public class Timeout : Name<int>
         {
         }
 
-        [NamedParameter("Retry times", defaultValue: "5")]
+        [NamedParameter("Retry times", defaultValue: "500")]
         public class RetryCount : Name<int>
```

Comment:
The number is a named parameter that can be configured. I have changed it to be internal sealed. And added comments.